client/state/themes/reducer.js

```diff
@@ -309,7 +309,14 @@ export const queries = ( () => {
 
 	return createReducer( {}, {
 		[ THEMES_REQUEST_SUCCESS ]: ( state, { siteId, query, themes, found } ) => {
-			return applyToManager( state, siteId, 'receive', true, themes, { query, found } );
+			return applyToManager(
+				state,
+				siteId,
+				'receive',
+				true,
+				themes,
+				query ? { query, found } : {}
+			);
```

Comment:
Is it possible that we don't actually need this change? I think [this](https://github.com/Automattic/wp-calypso/pull/11782/commits/6a6461a31c04887038cfc57caa669000d5798b1d#diff-dd8c5284a85f9baf309a0ec3a482d228R105) is enough.